src/main/java/edu/ucsb/cs156/courses/controllers/PSCourseController.java

```diff
@@ -149,6 +149,9 @@ public ArrayList<PSCourse> postCourses(
 	    hasSecondary = false;
 	}
 
+    if (coursesRepository.findByPsIdAndEnrollCd(psId, enrollCdPrimary).isPresent()) {
```

Comment:
It's just embarrassing to see indentation issues this blatant in a PR that you are asking to merge into main.

Have we not taught you better than this .. not just in this course, but in all of your UCSB coding courses up to this point?